api/src/pcapi/routes/backoffice/offerers/validation_blueprint.py

```diff
@@ -668,14 +686,19 @@ def get_reject_user_offerer_form(user_offerer_id: int) -> utils.BackofficeRespon
 
     form = offerer_forms.OptionalCommentForm()
 
-    return render_template(
-        "components/turbo/modal_form.html",
-        form=form,
-        dst=url_for("backoffice_web.validation.reject_user_offerer", user_offerer_id=user_offerer.id),
-        div_id=f"reject-modal-{user_offerer.id}",  # must be consistent with parameter passed to build_lazy_modal
-        title=f"Rejeter le rattachement à {user_offerer.offerer.name.upper()}",
-        button_text="Rejeter le rattachement",
-    )
+    kwargs = {
+        "form": form,
+        "dst": url_for("backoffice_web.validation.reject_user_offerer", user_offerer_id=user_offerer.id),
+        "div_id": f"reject-modal-{user_offerer.id}",  # must be consistent with parameter passed to build_lazy_modal
+        "title": f"Rejeter le rattachement à {user_offerer.offerer.name.upper()}",
+        "button_text": "Rejeter le rattachement",
+    }
+    if utils.is_request_from_htmx():
+        return render_template(
+            "components/dynamic/modal_form.html", target_id=f"#user-offerer-row-{user_offerer_id}", **kwargs
+        )
+
+    return render_template("components/turbo/modal_form.html", **kwargs)
```

Comment:
Proposition (comme ça on a l'essentiel au même endroit) 
```suggestion
    target_kwarg = {}
    if utils.is_request_from_htmx():
        target_kwarg = {"target_id": f"#user-offerer-row-{user_offerer_id}"}
    return render_template(
        "components/turbo/modal_form.html",
        form=form,
        dst=url_for("backoffice_web.validation.reject_user_offerer", user_offerer_id=user_offerer.id),
        div_id=f"reject-modal-{user_offerer.id}",  # must be consistent with parameter passed to build_lazy_modal
        title=f"Rejeter le rattachement à {user_offerer.offerer.name.upper()}",
        button_text="Rejeter le rattachement",
        **target_kwarg
    )
```

Reply:
My bad là dessus